### ISSUE TITLE
Persitent archiving of images and/or recipes

### ISSUE THREAD

**Issue by pauleve** [colomoto/colomoto-docker#78]
In addition to mirror on github packages, we should consider persistent archiving of images.

Currently, I see two approaches:
* backup the full images on Zenodo: quite easy to realize (docker can export the full image as a tar file), although I'm not sure how to automatize that. We can then add an option to the `colomoto-docker` program to fetch an image from zenodo
* alternatively/in parallel, ensuring that all images can be built over time (#76) sounds more lightweight (note that zenodo allows up to 50GB per record, so we are far from this limit). The main issue is then to have a persistent backup of conda packages, as they may be deleted or moved to other labels by their maintainers. A first step would be to have a `colomoto-history` conda channel and copy all packages there to ensure we can fetch them later. But then, I don't know how reliable is anaconda.org over time.

**Comment by pauleve** [comment]
Here is the doc to automate the upload on zenodo: https://developers.zenodo.org/#quickstart-upload
I can implement that during September, it sounds the most straightforward way to reliably archive our images.

**Comment by aurelien-naldi** [comment]
Archiving on zenodo sounds like a great idea!
Maybe we should tag some images as "long term" to reduce a bit the load?

**Comment by pauleve** [comment]
Superseeded by #141 for archiving with zenodo and #92  for building (latter has a very low priority)
